test/rekt/new_trigger_filters_test.go

```diff
@@ -42,56 +41,6 @@ func TestMTChannelBrokerNewTriggerFilters(t *testing.T) {
 		k8s.WithEventListener,
 		environment.Managed(t),
 	)
-	brokerName := "default"
-
-	env.Prerequisite(ctx, t, broker.InstallMTBroker(brokerName))
-	env.TestSet(ctx, t, newfilters.FiltersFeatureSet(brokerName))
-}
-
-func TestMTChannelBrokerAnyTriggerFilters(t *testing.T) {
-	t.Parallel()
-
-	ctx, env := global.Environment(
-		knative.WithKnativeNamespace(system.Namespace()),
-		knative.WithLoggingConfig,
-		knative.WithTracingConfig,
-		k8s.WithEventListener,
-		environment.Managed(t),
-	)
-	brokerName := "default"
-
-	env.Prerequisite(ctx, t, broker.InstallMTBroker(brokerName))
-	env.Test(ctx, t, newfilters.AnyFilterFeature(brokerName))
-}
-
-func TestMTChannelBrokerAllTriggerFilters(t *testing.T) {
-	t.Parallel()
-
-	ctx, env := global.Environment(
-		knative.WithKnativeNamespace(system.Namespace()),
-		knative.WithLoggingConfig,
-		knative.WithTracingConfig,
-		k8s.WithEventListener,
-		environment.Managed(t),
-	)
-	brokerName := "default"
-
-	env.Prerequisite(ctx, t, broker.InstallMTBroker(brokerName))
-	env.Test(ctx, t, newfilters.AllFilterFeature(brokerName))
-}
-
-func TestMultipleSinksMultipleTriggers(t *testing.T) {
-	t.Parallel()
-
-	ctx, env := global.Environment(
-		knative.WithKnativeNamespace(system.Namespace()),
-		knative.WithLoggingConfig,
-		knative.WithTracingConfig,
-		k8s.WithEventListener,
-		environment.Managed(t),
-	)
-	brokerName := "default"
 
-	env.Prerequisite(ctx, t, broker.InstallMTBroker(brokerName))
-	env.Test(ctx, t, newfilters.MultipleTriggersAndSinksFeature(brokerName))
+	env.TestSet(ctx, t, newfilters.NewFiltersFeatureSet(broker.InstallMTBrokerIntoFeature))
```

Comment:
Maybe we can use "parallel test set" ?

Reply:
Cool! I didn't know that was a thing :) 

I'll change that this morning